### PR TITLE
[vault-token-lp-balance] Adding farmersonly-fox-in-vaults strategy for FarmersOnlyFi DAO

### PR DESCRIPTION
**Pull request by FoxTheFarmer**
Changes proposed in this pull request:
- Adding farmersonly-fox-in-vaults strategy for FarmersOnlyFi DAO

### CODE REVIEW

File: src/strategies/farmersonly-fox-in-vaults/index.ts [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+  const lpFoxBalancesCalls = multicall(
+    network,
+    provider,
+    bep20Abi,
+    vaultFoxPools.map((vault) => [foxAddress, 'balanceOf', [vault.lpAddress]]),
+    { blockTag }
+  );
+
+  const userVaultFoxBalances: any = vaultFoxPools.map(async (vault) => {
```
Comment (ChaituVR) [inline src/strategies/farmersonly-fox-in-vaults/index.ts:91]: @FoxTheFarmer I see more than 10 multicall in this strategy, but we have a limit for this
```
There should be a maximum of 5 requests, a request can use "fetch" a "subgraphRequest" or "multicall".
```
https://github.com/snapshot-labs/snapshot-strategies#checklist-for-adding-a-new-strategy


Please check if there is an alternate way to get this :)

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/single-staking-vault-balanceof/index.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    addresses.map((address: any) => [
+      options.vaultAddress,
+      'wantLockedTotal',
+      [address]
+    ]),
+    { blockTag }
+  );
+
+  const vaultBalances = await Promise.all([vaultBalancesCalls]);
+  console.log('vaultBalances', vaultBalances);
```
Comment (ChaituVR) [inline src/strategies/single-staking-vault-balanceof/index.ts:32]:
```suggestion
```

File: src/strategies/vault-token-lp-balance/index.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      options.vaultChefAddress,
+      'stakedWantTokens',
+      [options.pid, address]
+    ]),
+    { blockTag }
+  );
+
+  const vaultBalances = await Promise.all([userVaultLpBalanceCalls]);
+
+  console.log(vaultBalances);
```
Comment (ChaituVR) [inline src/strategies/vault-token-lp-balance/index.ts:55]:
```suggestion
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Thanks @FoxTheFarmer  Just remove these logs, everything else looks good to go

**Comment by ChaituVR** [comment]
@FoxTheFarmer Please resolve conflicts

**Review by bonustrack — APPROVED** [review]

**Pull request merged**